src/main/java/com/autotune/common/data/dataSourceQueries/DataSourceQueries.java

```diff
@@ -9,7 +9,11 @@ public class DataSourceQueries {
     public enum PromQLQuery {
         NAMESPACE_QUERY("sum by (namespace) ( avg_over_time(kube_namespace_status_phase{namespace!=\"\" ADDITIONAL_LABEL}[15d]))"),
         WORKLOAD_INFO_QUERY("sum by (namespace, workload, workload_type) ( avg_over_time(namespace_workload_pod:kube_pod_owner:relabel{workload!=\"\" ADDITIONAL_LABEL}[15d]))"),
-        CONTAINER_INFO_QUERY("sum by (container, image, workload, workload_type, namespace) ( avg_over_time(kube_pod_container_info{container!=\"\" ADDITIONAL_LABEL}[15d]) * on (pod, namespace) group_left(workload, workload_type) avg_over_time(namespace_workload_pod:kube_pod_owner:relabel{workload!=\"\" ADDITIONAL_LABEL}[15d]))");
+        CONTAINER_INFO_QUERY("sum by (container, image, workload, workload_type, namespace) (" +
+                "  avg_over_time(kube_pod_container_info{}[15d]) *" +
```

Comment:
Doesn't it need ADDITIONAL LABEL ?

Reply:
Oh good catch

Reply:
@msvinaykumar I still don't see ADDITIONAL_LABEL added, can you check.

Reply:
Done